build.gradle.kts

```diff
@@ -7,7 +7,7 @@ plugins {
   id("uk.gov.justice.hmpps.gradle-spring-boot") version "6.0.7"
   kotlin("plugin.spring") version "2.0.21"
   kotlin("plugin.jpa") version "2.0.21"
-  id("org.jetbrains.kotlinx.kover") version "0.9.0-RC"
+  id("org.jetbrains.kotlinx.kover") version "0.8.3"
```

Comment:
This a lower?

Reply:
Bit worry to use RC. Im more happy to stick with stable version